site/tests/universal/page-header.test.ts

```diff
@@ -1,9 +1,33 @@
 import { expect, test } from "@playwright/test";
+import type { Page } from "playwright";
+
+const openMobileNav = async (page: Page) => {
+  if (await page.locator("[data-testid='mobile-nav']").isVisible()) {
+    return;
+  }
+  await expect(
+    page.locator("[data-testid='mobile-nav-trigger']"),
+  ).toBeVisible();
+  await page.locator("[data-testid='mobile-nav-trigger']").click();
+  await expect(page.locator("[data-testid='mobile-nav']")).toBeVisible();
+};
+
+test("page header navigation works", async ({ page, isMobile }) => {
+  if (isMobile) {
+    test.skip();
+  }
```

Comment:
I assume that this is a temporary statement because there is a ternary expression below

Reply:
Yes... That's what I was referring to here => https://github.com/blockprotocol/blockprotocol/pull/444#discussion_r924520450